src/MatrixFields/MatrixFields.jl

```diff
@@ -88,6 +88,8 @@ const ColumnwiseBandMatrixField{V, S} = Fields.Field{
 } where {
     V <: AbstractData{<:BandMatrixRow},
     S <: Union{
+        Spaces.AbstractSpectralElementSpace,
+        Spaces.PointSpace,
         Spaces.FiniteDifferenceSpace,
         Spaces.ExtrudedFiniteDifferenceSpace,
         Operators.PlaceholderSpace, # so that this can exist inside cuda kernels
```

Comment:
I think this is all the spaces we have, so let's just do `S <: Spaces.AbstractSpace` here.